nucypher/crypto/powers.py

```diff
@@ -92,37 +92,39 @@ class BlockchainPower(CryptoPowerUp):
     """
     not_found_error = NoBlockchainPower
 
-    def __init__(self, blockchain: 'Blockchain', account: str) -> None:
+    def __init__(self, client=None, device=None) -> None:
         """
         Instantiates a BlockchainPower for the given account id.
         """
-        self.blockchain = blockchain
-        self.account = account
```

Comment:
I don't like this new take on `BlockchainPower`. Since it's no account-specific anymore, is it really supposed to give you power over *all the accounts* managed by the blockchain client? The previous `BlockchainPower` was consistent with `SigningPower` and `DecryptingPower`, in the sense that it empowered a Character with the potential conferred by a single private key (either a ECDSA-signing key, a decryption key or an ETH key). 

I agree that the name "BlockchainPower" is confusing. "Blockchain" is not an action, as opposed to "Signing" or "Decrypting". Maybe this is related to the concept of "TransactingPower"?

Reply:
@cygnusv - Note the comment in the PR description: 

> Note: Calls to BlockchainPower or TransactingPower can be considered concept-code, for that, see PR #1092

Reply:
Resolved in 11e79d0 :heavy_check_mark: